tests/test_hex.py

```diff
@@ -0,0 +1,37 @@
+import sys
+from collections import OrderedDict
+from typing import Any
+from unittest.mock import Mock
+
+import hypothesis.extra.numpy as hnp
+import hypothesis.strategies as st
+import numpy as np
+from hypothesis import given
+
+sys.modules["mujoco_py"] = Mock()
+from opthex.hex import HexState  # isort:skip
+
+
+@st.composite
+def hex_state_strategy(draw: Any) -> HexState:
+    """Generate a random HexState object"""
+    joint_names = draw(st.lists(st.text(), unique=True))
+    num_joints = len(joint_names)
+    joint_pos = draw(hnp.arrays(dtype=np.float32, shape=num_joints))
+    joint_vel = draw(hnp.arrays(dtype=np.float32, shape=num_joints))
+    acceleration = draw(hnp.arrays(dtype=np.float32, shape=3))
+    return HexState(
+        qpos=OrderedDict(zip(joint_names, joint_pos)),
+        qvel=OrderedDict(zip(joint_names, joint_vel)),
+        acceleration=acceleration,
+    )
+
+
+@given(hex_state=hex_state_strategy())
+def test_hex_state_joint_state(hex_state: HexState) -> None:
```

Comment:
just wondering: is there an advantage of using hypothesis instead of native pytest fixtures?

Reply:
Hypothesis will generate a variety of different test cases (100 by default) and will search for edge cases automatically so you won't have to think about it. It also has some other fancy features like giving you the simplest failure case it can find. In this case, it's probably overkill but could be useful later when we have more tools that consume the state.